### ISSUE TITLE
study groups

### ISSUE THREAD

**Issue by max-mapper** [internet3000/ideas#1]
I am interested in designing a meetup format/community centered around the idea of study groups with a small cohort and mentor, with a open source twist.

The main goal is to create two things: high quality free curriculum and an 'open community' of open source educators/learners (meaning it's not all on e.g. meetup.com)

Mozilla Science Lab is working on something similar (I gave them some ideas and they ran with it): https://github.com/mozillascience/studyGroup

Mozilla's audience is very specifically people in the 'open science' community, and it isn't really targeted at general open source people.

A recent success I've helped design is NodeSchool Chapters (https://github.com/nodeschool/organizers#how-to-start-a-new-nodeschool-chapter). We have 120+ chapters around the world now.

Right now the only events that NodeSchool does are "planned" events. That is, the organizer has to find a venue, get the word out, set up RSVPs, find mentors, find food + possibly sponsors. It's a lot of work, and they only happen every month or two.

Another one worth mentioning is http://cyber.wizard.institute/

Ideally I wouldn't want these study groups to be JS or Node specific. I'd rather they be one step removed, so about open source or the web, and not tied to one specific tool or silo. Instead, a theoretical study group site could have different recommended curriculums to progress through as a study group, and NodeSchool would be one of them, but we could recommend other curriculum as well (like stuff from railsbridge, pyladies, opentechschool etc).

The overall main goal is to build a peer learning and mentorship network, bootstrapped by the nodeschool international community but one that is wider in scope than just node.

Mentorship is hard, but so far I've had pretty good success building a network of existing professionals who wanna give back to the community, so I think it's worth a shot :)

Including some feedback I've received:

> on a high level, i think this is really powerful. one of the big selling points of coding bootcamps is being around others to help you out and motivate you. There's actually a very tight knit alumni community that do a lot to help each other out. But not everyone can afford that or quit their current job to do that. If we can do this grassroots, that's huge. I don't think it'll replace formal learning completely - some beginners are less self-directed and still require a mentor or teacher, but those who are self-directed learners could benefit a lot.
> 
> related thought: i wonder if the target learners should be novices - those with the basic skill set but looking to level up. That's a huge gap that I don't think is being filled by any school or org that I know of. Learning from a peer group (coworkers or project collaborators) has been the most effective method for me personally as I moved from novice to intermediate. Same goes for trying out a new tool or framework.
> 
> creating a space where everyone feels welcome is essential. it's scary to ask strangers for help, less so in person that online, but it'll still be critical to making the group effective. This may not be a concern, but how do we make sure it doesn't turn into like, brogrammer hack night. :)
> 
> I think the recurring meetings is the way to go. Currently the nodeschool meetups are basically large one off study groups. I'd rather make the study groups feel more like 0 planning, dependable (because of predictable scheduling e.g. every wednesday at 6pm at the same place) "just show up if you can" type of events.
> 
> I think events that are targeted at specific groups, e.g. PyLadies or BlackGirlsCode are very important, but at the same time need to be more broadly addressed at the top level. Maybe theres a way we can be seen as a sort of 'brand', so that if a specific subcommunity wanted to organize e.g. "PDX Ladies Study Group" then they could hack the process to keep it from getting drowned out in creepy dudes while still being compatible with the rest of the ecosystem.
> 
> A name that isn't toooooo generic ("study group" seems a little vague) but at the same time explains that it's a study group for aspiring coders would be awesome.

What do you think???

**Comment by ghost** [comment]
We've had a javascript study group in Oakland for 2 years and it's been fantastic. It organized itself for the most part after starting out as east bay javascript, but organizers got busy but the event stayed on the calendar and people keep showing up to hack on their projects and ask questions.

**Comment by yoshuawuyts** [comment]
Yeah, JS night at sudo is a great example! The first time I attended we ended up doing a WebRTC themed hack session and discussing everyone's [omega projects](http://mediterraneajs.eu/people/dominic_tarr.html). It had a very laid back atmosphere and I think it was useful for people of any skill level.

I agree with the sentiment that nodeschool sessions are hard to organize; self organizing study groups might be a good alternative approach.

**edit**: Something I've noticed with nodeschool / meetups in general is that they have a centralized setup; `organizers -> mentors -> students` which often falls apart once either organizers or mentors become absent. I imagine study groups to be peer-to-peer and thus more sturdy over time.

**Comment by yoshuawuyts** [comment]
Going to run an internet3000 hack night in Melbourne next week, no idea what will come of it. Lookahead Search is probably sponsoring location + food. Will report back once we've run it (if it all goes through, haha). 

**edit**: I have no idea if the description is correct. I figured I'd just take the idea and run with it to see what happens. If anyone's got ideas on how to improve wording / format / better channel the spirit of internet3k, please comment away.

Here's the description I wrote up:

---

# internet3000 - decentralized hack night

from [time], at [location]
- no talks
- no agenda
- bring your own project

Come together and hack on things! Whether you're new to coding or a seasoned programmer, making things is more fun when you're around other people.

Food + drinks are kindly provided by [sponsor] because y'know, making things is more fun when you're well-nourished.

## FAQ

### What is internet3000?

Internet3000 is about coming together, making things and learning from peers. There's no minimum skill level, just bring something to work on.

### Decentralized?

Most meetups rely on a centralized organization to organize and manage events. It's a lot of work for the organizers, and limits the amount of events that can be organized. With internet3000 the participants provide the content, making it much easier to set up.

### Sounds cool, can I run my own?

Yes, definitely. You're encouraged to run your own. Come together, make things and learn from each other. It's always hard to tell if an event will be a success in advance, but that shouldn't stop you from having fun trying!

## See Also
- https://github.com/internet3000/ideas
- https://github.com/mozillascience/studyGroup
- https://sudoroom.org/events/js-meeting-2015-07-30/
